video_mapper.go

```diff
@@ -326,7 +331,7 @@ func unsafeJSONMarshal(v interface{}) ([]byte, error) {
 	return b, nil
 }
 
-func isPublishEvent(video map[string]interface{}) (publishEvent bool, err error) {
+func isPublishEvent(video map[string]interface{}) (_ bool, err error) {
```

Comment:
I have added _ in first place due to IDEA compilation warning. Fixed this by removing usage of name return values.